plugins/modules/grafana_team.py

```diff
@@ -254,16 +254,17 @@ def grafana_switch_organisation(self, org_id):
     def organization_by_name(self, org_name):
         url = "/api/user/orgs"
         organizations = self._send_request(url, headers=self.headers, method="GET")
+
         try:
-            orga = next((org for org in organizations if org["name"] == org_name))
-            if orga:
-                return orga["orgId"]
+            return next(
```

Comment:
I mulled over this for quite a while, but this makes much more sense.

Reply:
The function comes from another module and I think even from me, but I only noticed the improvement here in this case... 😄